.github/dependabot.yml

```diff
@@ -1,5 +1,10 @@
 version: 2
 updates:
+- package-ecosystem: "npm"
+  directory: "/"
+  schedule:
+    interval: weekly
```

Comment:
I figured that was good enough for addons-server

Reply:
generally, but for updates to our own packages like the linter we usually like it quicker

Reply:
Good point

Reply:
e1620da134273d95cbb8733014934ae604039452